### PR TITLE
Fix README CMake option descriptions & CLI test coverage

### PR DESCRIPTION
**Pull request by marcluque**
The README had a wrong link for the sanitize build target.
Also add more CLI tests to reach full branch coverage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,10 +66,9 @@ Don't forget to give the project a star! Thanks again!
 1. Fork the Project
 2. Create your Feature Branch (`git checkout -b feature/cool-feature`)
 3. Implement your desired Feature!
-3. Commit your Changes (`git commit -am 'Add some cool-feature'`)
```
Comment (mormod) [inline README.md:69]: It should say "feat: add some cool feature description here!", as this matches the current commit style

File: README.md [hunk 2]
```diff
@@ -66,10 +66,9 @@ Don't forget to give the project a star! Thanks again!
...
-4. Push to the Branch (`git push origin feature/cool-feature`)
-5. Create a new [Pull Request](https://github.com/benzammour/yobemag/pulls)
-
+4. Commit your Changes (`git commit -am 'Add some cool-feature'`)
```
Comment (mormod) [inline README.md:69]: `git commit -a` can be a bit dangerous if one is working with IDEs, that none of us use.

File: README.md (hunk 1 above)
Comment (marcluque) [reply README.md:69]: I think so too. What do you say @Benzammour?

File: README.md (hunk 2 above)
Comment (marcluque) [reply README.md:69]: What alternative do you suggest? 🤔

File: README.md (hunk 1 above)
Comment (Benzammour) [reply README.md:69]: Yes, I agree!

**Review by mormod — APPROVED** [review]

**Pull request merged**